docs/topics/coalesce-swashbuckle.md

```diff
@@ -0,0 +1,102 @@
+# Coalesce Swashbuckle
+
```

Comment:
Lets update all references to "Swagger" to "OpenAPI" where appropriate, or to "Swashbuckle" when referring to the specific nuget packages.

Swagger is a suite of tools and services created and sold by SmartBear around the OpenAPI standard. We should include maybe a short blurb in the first paragraph for searching purposes, but nothing else aside from the references in Swashbuckle's method names (which are named so for legacy/compatibility reasons). 

Something like "... OpenAPI (sometimes referred to as its pre-2015 name "Swagger") ..."